src/Umbraco.Web.UI.Client/test/unit/app/templates/template-editor-controller.spec.js

```diff
@@ -47,6 +47,7 @@
                     getCursorPosition: function() {},
                     getValue: function() {},
                     setValue: function() {},
+                    focus: function() {},
```

Comment:
Is this just a dummy test or do we have a proper matching test for this @madsrasmussen ?